frontend/app/error.vue

```diff
@@ -34,29 +51,44 @@ const errorMessage = computed(() => {
   <NuxtLayout name="default">
     <div class="w-full mx-10 flex content-center items-center justify-center flex-col-reverse md:flex-row">
       <div class="flex flex-col md:mr-10 max-w-3/4 md:max-w-3/5 space-y-5">
-        <h1 class="text-balance text-8xl text-primary font-bold">
-          {{ error?.statusCode }}
-        </h1>
-        <h2 class="whitespace-pre-line text-2xl text-white">
-          {{ errorMessage }}
-        </h2>
-        <LazyUModal title="Więcej informacji o błędzie:" hydrate-on-visible>
-          <UButton label="Więcej informacji..." variant="outline" class="w-fit mt-4" />
+        <template v-if="showTimer">
```

Comment:
1. I am not sure if this is good idea performance wise, because I don't think nuxt cannot treat `<templaate>` as lazy component
2. Code doesn't look really good because in one file you handle logic for normal errors and for errors with timers,

You could just create separate components for those 2 types

Reply:
For example if this is normal 404 error you still send requests to get event status